### PR TITLE
Private cloud preview component

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/sections/view-private-cloud/macro.test.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+        expect(requiresHSCNInnerComponent.length).toEqual(1);
+        expect(requiresHSCNInnerComponent.text().trim()).toEqual('This Solution requires a HSCN/N3 connection');
+
+        done();
+      });
+  });
+
+  it('should not render the requires HSCN answer if not given', (done) => {
+    const context = {
```
Comment (kat1906) [inline app/sections/view-private-cloud/macro.test.js]: I think you also need to add tests for when the model, link and summary are not provided as all fields are optional

Comment (alibombaye) [reply app/sections/view-private-cloud/macro.test.js]: agree with Katie. We should test for these possibilities too

File: app/sections/view-private-cloud/macro.test.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+          .find('div[data-test-id="view-section-table-row-component"]')
+          .find('[data-test-id="view-question-data-text-link"]');
+
+        expect(linkInnerComponent.length).toEqual(1);
+        expect(linkInnerComponent.text().trim()).toEqual('www.healthcare-pro.co.uk/healthcare-system-1/hybrid-hosting');
+
+        done();
+      });
+  });
+
```
Comment (alibombaye) [inline app/sections/view-private-cloud/macro.test.js]: I would like a test to make sure the row containing the `summary` and `link` is not rendered at all when both of these answers are not provided.

File: app/sections/view-private-cloud/macro.test.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const requiresHSCNQuestionRow = $('[data-test-id="view-section-table-row-summary"]');
```
Comment (alibombaye) [inline app/sections/view-private-cloud/macro.test.js]: I think this const name needs updating? should it be `summaryRow`?

Comment (BenBJSS) [reply app/sections/view-private-cloud/macro.test.js]: Updated

**Pull request merged**